src/utils/data/index.js

```diff
@@ -63,6 +67,18 @@ const getBar = (percentage) => {
   return bar;
 };
 
+const getPads = (n) => {
+  if (deviceInfo.osName === "Linux") {
+    if (browserInfo.isChrome) {
+      return FIGURE_SPACE.repeat(n);
```

Comment:
@GuiLeme , can you try change it either to
return FULL_BLOCK.repeat(n);
or
return EN_SPACE.repeat(n);
and check if your problem persists?

Reply:
Hi @GuiLeme , in my Ubuntu box (22.04) it did not work. With Brave browser it look like:
![image](https://github.com/bigbluebutton/bbb-playback/assets/45039819/a0cf8e83-8e43-4b01-b91d-62b6ce5bfa37)

It could be a locale problem (though I am using English).... let me leave it here and wait for bug reports.

Reply:
@hiroshisuga, Hum, yeah, there must be a problem with locale, I don't think mine is set to English. But as I said, I guess it is just fine the way it is. I don't think this should be a blocking problem.

Let's see what @antobinary thinks about this. I guess everything is fine! If everyone agrees, I can approve the PR!